physics/sgscloud_radpre.F90

```diff
@@ -35,16 +35,19 @@ module sgscloud_radpre
 !!
 !>\section sgscloud_radpre_mod  SGS Cloud Scheme Pre General Algorithm
       subroutine sgscloud_radpre_run(    &
```

Comment:
Not that this matters for this PR, but RRTMGP doesn't use sgscloud_radpre_run, so these change will ultimately needed to be ported to [GP's cloud coupling](https://github.com/ufs-community/ccpp-physics/blob/ufs/dev/physics/GFS_rrtmgp_cloud_mp.F90).

Looking at this now it seems that they diverged sometime ago, so nbd.

Reply:
Yes, I'd like this to be cleaned up. Done once for all configurations. Unfortunately, I need a lot of time to soak up what we have and know best what to propose in it's place.